docs/pages/management/diagnostics/reference/configuration.mdx

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I think as long as these fields are in the main configuration reference, we can remove this guide. It might get confusing to have reference docs in the `diagnostics` section _and_ in the `reference` section.